add-on/preferences/prefs.js

```diff
@@ -34,3 +33,6 @@ pref("loop.facebook.enabled", true);
 #else
 pref("loop.facebook.enabled", false);
 #endif
+pref("loop.facebook.appId", "1519239075036718");
+pref("loop.facebook.shareUrl", "https://www.facebook.com/dialog/send?app_id=%APP_ID%&link=%ROOM_URL%&redirect_uri=%REDIRECT_URI%");
```

Comment:
Having a separate `appId` pref seems unnecessary when `shareUrl` can just as easily be changed to have a different value/`app_id=1234`